Config/Tests.xcconfig

```diff
@@ -10,4 +10,9 @@ WARNING_CFLAGS = -Wno-objc-messaging-id
 
 OTHER_CFLAGS = $(inherited) -iframework "$(SRCROOT)/../Vendor/$(APPCENTER_BUILD_PLATFORM)/OCMock" -iframework "$(SRCROOT)/../Vendor/$(APPCENTER_BUILD_PLATFORM)/OCHamcrest" -iframework "$(SRCROOT)/../Vendor/$(APPCENTER_BUILD_PLATFORM)/OHHTTPStubs" -iframework "$(PLATFORM_DIR)/Developer/Library/Frameworks"
 
+OTHER_LDFLAGS = $(inherited) -ObjC -lz
```

Comment:
Why do you use `$(inherited)` here instead of `$(OTHER_LDFLAGS)`like in other places?